.github/workflows/general.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Suggested to add `3.13` as current main

Reply:
Marking as resolved as it has been addressed.

Reply:
Done